### PR TITLE
Added functionality to add/delete multiple songs

### PR DESCRIPTION
**Pull request by kyliepedersen**
- Made commit function that has to be called when ready to commit changes in playlist to Spotify. 
- Add and delete functions work the same way, but without committing to playlist every time.

### CODE REVIEW

File: backend/gitSound.py [hunk 1]
```diff
@@ -191,28 +191,30 @@ def removeSongFromPlaylist(self, pid, songid):
...
+        self.repo.index.read()
+        self.repo.index.add("index.txt")
+        self.repo.index.write()
 
-        # commit
-        repo.create_commit("HEAD", self.author, self.comitter,
-                           "Removed " + songid, tree, [repo.head.target])
+        # commit changes to playlist
+        self.repo.create_commit("HEAD", self.author, self.comitter,
+                           "Changes committed to " + pid, tree, [self.repo.head.target])
```
Comment (mguida22) [inline backend/gitSound.py:217]: We might want to say something different here so we don't always have the same commit messages.. not sure what that would be though

Comment (kyliepedersen) [inline backend/gitSound.py:217]: I agree. I put it in there as a placeholder for now. If you think of something better, we should change it. Right now we are not keeping a list of all changes between commits, so there isn't a way to give feedback on all of the changes

Comment (mguida22) [inline backend/gitSound.py:217]: :+1:

**Comment by mguida22** [comment]
This looks good to me, do we need the get/set in python though?

**Comment by mguida22** [comment]
Closes #9

**Pull request merged**